### PR TITLE
OCM-6448 | fix: Fix UX bugs with CLI

### PR DESCRIPTION
**Pull request by hunterkepley**
Fixes these UX issues:
https://issues.redhat.com/browse/OCM-7896
https://issues.redhat.com/browse/OCM-7885

### CODE REVIEW

File: cmd/rosa-support/create/sg/cmd.go [hunk 1]
```diff
@@ -51,7 +51,7 @@ func init() {
 		&args.count,
 		"count",
 		"",
-		0,
+		1,
```
Comment (gdbranco) [inline cmd/rosa-support/create/sg/cmd.go:54]: Is this change expected or should the change be on the example to include --count?

Comment (hunterkepley) [reply cmd/rosa-support/create/sg/cmd.go:54]: yes, with `0`, it will not create a SG, due to this value being used in the loop directly

Comment (hunterkepley) [reply cmd/rosa-support/create/sg/cmd.go:54]: if you do not set `--count` yourself, QE does not have this required in their normal flow though

File: cmd/rosa-support/create/sg/cmd.go [hunk 2]
```diff
@@ -51,8 +51,8 @@ func init() {
 		&args.count,
 		"count",
 		"",
-		0,
-		"Additional number of security groups to be created for the vpc",
+		1,
+		"Number of security groups to be created for the vpc",
```
Comment (gdbranco) [inline cmd/rosa-support/create/sg/cmd.go]: I think this should be moved back

**Review by gdbranco — APPROVED** [review]

**Pull request merged**